django/db/utils.py

```diff
@@ -26,6 +26,10 @@ class DatabaseError(Error):
     pass
 
 
+class NoRowsAffected(DatabaseError):
+    pass
```

Comment:
I feel like it should live next to `ObjectDoesNotExist` / `MultipleObjectsReturned` as it's virtually the same concept

Maybe something like `NoObjectAffected` (that inherits from `DataBaseError` for unfortunate historical reasons)

Reply:
I'm having issues placing `ObjectNotUpdated` next to `ObjectDoesNotExist` because I have to import `DataBaseError` and I run into circular import issues.

Any idea how to overcome it?

Reply:
@EnriqueSoria the `ObjectNotUpdated` class defined in `django.core.exception` doesn't have to subclass `DataBaseError` so there should not be any circular import errors. Only [the subclasses](https://github.com/django/django/pull/18244#pullrequestreview-2106497667) defined in `models.base` should extent both and `DataBaseError` is already available there.

Reply:
That makes a lot of sense, thanks!

Reply:
This can be removed now @EnriqueSoria as it's superseded by `ObjectNotUpdated`.